docs/docusaurus/docs/cloud/data_sources/manage_data_sources.py

```diff
@@ -0,0 +1,60 @@
+"""
+This is an example script for how to edit a Data Source
+
+To test, run:
+pytest --docs-tests -k "cloud_docs_manage_data_sources" tests/integration/test_script_runner.py
+"""
+
+# EXAMPLE SCRIPT STARTS HERE:
+# <snippet name="docs/docusaurus/docs/cloud/data_sources/manage_data_sources.py - full sample">
+# Create a Data Context object.
+# <snippet name="docs/docusaurus/docs/cloud/data_sources/manage_data_sources.py - get cloud context">
+import great_expectations as gx
+
+context = gx.get_context(mode="cloud")
+# </snippet>
+
+# Verify that you have a GX Cloud Data Context.
+# <snippet name="docs/docusaurus/docs/cloud/data_sources/manage_data_sources.py - verify context type">
+print(type(context).__name__)
+# </snippet>
+# Hide this
+data_source_name = "S3 Data Source"
+# Hide this
+bucket_name = "my-bucket"
+# Hide this
+boto3_options = {
+    # Hide this
+    "aws_access_key_id": "${MY_S3_KEY_ID}",
```

Comment:
making this match other similar instances so only one env var has to be set for CI



```suggestion
    "aws_access_key_id": "${S3_KEY_ID}",
```